pkg/sql/run_control_test.go

```diff
@@ -365,6 +363,8 @@ GRANT admin TO has_admin2;
 			}
 		}()
 	}
+	// Give the cancel queries a chance to propagate before stopping the cluster.
+	time.Sleep(time.Second)
```

Comment:
interesting, i thought `CANCEL QUERY` would only return success if the query had already been canceled. i guess not.

optional suggestion: to make this test a bit faster, we could move this sleep to be right after line 360 instead of here.